### ISSUE TITLE
Plug & Play is not supported on Mac OS X

### ISSUE THREAD

**Issue by LudovicRousseau** [cdesiles/node-pcsc#6]
```
./example.js 

/Users/rousseau/Documents/github/node-pcsc/lib/node-pcsc.js:22
var pcsc =  new PCSC();
            ^
TypeError: PnP is not supported
```

`\\?PnP?\Notification` is not (yet) supported on Mac OS X.

You can have a look at my (work in progress, not working) mac branch at https://github.com/LudovicRousseau/node-pcsc/tree/mac

**Comment by cdesiles** [comment]
Is this solved in the last 0.2.2 version?

**Comment by LudovicRousseau** [comment]
yes.
